src/app/stake/btc/page.tsx

```diff
@@ -1,86 +1,192 @@
 "use client";

```

Comment:
non-blocking: my preference would be that we create a server component here that fetches the first page of finality providers, which gets passed into the client component.

```
async function Page() {
  try {
      const finalityProviders = await getFinalityProviders({ page: 1 });
      return <StakeBtc providers={finalityProviders} />;
  } catch() { return <Error /> }
}
```

`providers` would be set as initialData in useInfiniteQuery and initialPageParams would be set to page=1